modules/management-console-agent/src/main/java/org/apache/ignite/agent/utils/AgentUtils.java

```diff
@@ -64,7 +65,7 @@ private AgentUtils() {
      * @param clusterId Cluster ID.
      */
     public static String monitoringUri(String srvUri, UUID clusterId) {
-        return srvUri + "/clusters/" + clusterId + "/monitoring-dashboard";
+        return URI.create(srvUri + "/clusters/" + clusterId + "/monitoring-dashboard").normalize().toString();
```

Comment:
Lets create utility method for this?

Reply:
Utility method for utility method? :) I think this method will only be used to monitoring URI.